packages/db/src/project/test/compilationSources/truffle-config.js

```diff
@@ -1,13 +1,21 @@
+const Ganache = require("ganache");
+
+const provider = Ganache.provider({
+  // note instamine must be set to eager (default) with vmErrorsOnRPCResponse enabled
+  vmErrorsOnRPCResponse: true,
+  logging: {
+    quiet: true
+  }
+});
```

Comment:
Nit/confustion: the comment is confusing because it implies a code action and mentions the default, which nullifies the purpose of the comment? Perhaps explicitly setting miner.instamine = 'strict' with a comment to explain the pairing would be better?